package avalon.network;

message GameSettings {
	required int32 players = 1;
	required bool merlin = 2;
	required bool percival = 3;
	required bool mordred = 4;
	required bool morgana = 5;
	required bool assassin = 6;
	required bool oberon = 7;
}
